frontend/src/app/components/homescreen/blocks/new-features.component.ts

```diff
@@ -26,13 +26,11 @@
 // See doc/COPYRIGHT.rdoc for more details.
 // ++
 
-import {Component, Injector} from '@angular/core';
+import {Component} from '@angular/core';
 import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
-import {boardTeaserWebsiteURL} from "core-app/modules/boards/board-constants.const";
-import {OpModalService} from "core-components/op-modals/op-modal.service";
-import {BoardVideoTeaserModalComponent} from "core-app/modules/boards/board/board-video-teaser-modal/board-video-teaser-modal.component";
 import {DomSanitizer} from "@angular/platform-browser";
+import {dashboardWebsiteUrl} from "core-app/modules/dashboards/dashboard-constants.const";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/be1916a942caec25c3d384ac3746caf4b0648df1/frontend/src/app/components/homescreen/blocks/new-features.component.ts#L34"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>